Description: generate pkg-config files for libACE_INet and libACE_INet_SSL
 They are missing upstream.
Forwarded: needed
Author: Pau Garcia i Quiles <pgquiles@elpauer.org>
Last-Update: 2011-02-19

--- a/protocols/ace/INet/ACE_INet.pc.in
+++ b/protocols/ace/INet/ACE_INet.pc.in
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: ACE_INet
+Description: ACE INet Client Protocols Library
+Requires: ACE
+Version: @VERSION@
+Libs: -L${libdir} -lACE_INet
+Cflags: -I${includedir}
--- a/protocols/ace/INet/ACE_INet_SSL.pc.in
+++ b/protocols/ace/INet/ACE_INet_SSL.pc.in
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: ACE_INet_SSL
+Description: ACE INet Client Protocols With SSL Library
+Requires: ACE_SSL
+Version: @VERSION@
+Libs: -L${libdir} -lACE_INet_SSL
+Cflags: -I${includedir}
--- a/protocols/ace/INet/inet.mpc
+++ b/protocols/ace/INet/inet.mpc
@@ -54,6 +54,11 @@ project(INet) : acelib, ace_output, inst
   specific {
     install_dir = ace/INet
   }
+
+  Pkgconfig_Files {
+    ACE_INet.pc.in
+  }
+
 }
 
 // This project will be generated with and without ssl=1 feature
--- a/protocols/ace/INet/inet_ssl.mpc
+++ b/protocols/ace/INet/inet_ssl.mpc
@@ -25,4 +25,9 @@ project(INet_SSL) : acelib, ace_output,
   specific {
     install_dir = ace/INet
   }
+
+  Pkgconfig_Files {
+    ACE_INet_SSL.pc.in
+  }
+
 }
